proto/buf.yaml

```diff
@@ -0,0 +1,27 @@
+# Generated by "buf config migrate-v1beta1". Edit as necessary, and
+# remove this comment when you're finished.
+#
+# This module represents the "proto" root found in
+# the previous configuration.
+version: v1beta1
+name: buf.build/cosmos/gaia
+deps:
+  - buf.build/cosmos/cosmos-sdk
```

Comment:
You cannot do that just yet, the SDK protos are not in the registry for v0.45.
This will use the latest from the registry, which is main.
Consumers of our protos should pin the version. This was done by @yaruwangway here btw: https://github.com/cosmos/gaia/pull/1766

Reply:
ahhh thanks a ton!